### PR TITLE
Clean up clipboard handling

### PR DESCRIPTION
**Pull request by crizzitello**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

- Remove the clipboardhelper as each method was used only once in the application.
- Rename Gui Related Codeblock vars
  - Rename: `AppConfig::captureCodeblockShortcut` -> `AppConfig::captureClipboardShortcut`
  - Rename `Settings::recordCodeblockShortcutTextBox` -> `Settings:captureClipboardShortcutTextBox`
  - Rename `HotkeyManager::ACTION_CAPTURE_CODEBLOCK` -> `HotkeyManager::ACTION_CAPTURE_CLIPBOARD`
  - Rename `HotkeyManager::codeblockHotkeyPressed` -> `HotkeyManager::clipboardHotkeyPressed`
  - Rename: `TrayManager::onCodeblockCapture` -> `TrayManager::onClipboardCapture`
  - Rename: `TrayManager::captureCodeblockActionTriggered` -> `TrayManager::captureClipboardActionTriggered`
- Settings Dialog Change `Record Codeblock Shortcut` -> `Capture Clipboard Shortcut`
-  Tray Manager Action Text: `Add from Codeblock` -> `Capture from Clipboard`
-  Settings key for the codeblock shortcut now uses clipboard in name.
-  Clipboard Action behavior: Capture clipboard contents as evidence
    -  If the clipboard has text it will save as text and show the related dialog.
    -  If the clipboard is an image it will save that and trigger the related dialog

### CODE REVIEW

File: src/traymanager.cpp [hunk 1]
```diff
@@ -198,27 +200,40 @@ void TrayManager::captureAreaActionTriggered() {
...
+        QImage img = qvariant_cast<QImage>(mimeData->imageData());
+        img.save(path);
+        type = Screenshot::contentType();
     }
-    catch (QSqlError& e) {
-      std::cout << "could not write to the database: " << e.text().toStdString() << std::endl;
+
+    if (path.isEmpty() || type.isEmpty()) {
+        QTextStream(stdout) << "Error Gathering Evidence from clipboard" << Qt::endl;
+        return;
```
Comment (JoelAtDeluxe) [inline src/traymanager.cpp]: This is not equivalent to the try/catch, and probably needs a bit more work. There are a few things going on here, so:

* Most importantly, `createNewEvidence` can still throw an error if there's a problem writing to the internal database (sqlite). This isn't caught here, so I don't know what will happen. My assumption is that it'll manifest as a crash... Is that right?
   * The same is true for `saveCodeblock`. Though that error was never caught. Oops.
* `path` and `type`: both of these are set to definite values in the `if` and `else if` blocks, so they are reliably not-empty. So, this check isn't that helpful. Notably, even though these are set, they aren't necessarily valid. 
* Currently, if you try to record a codeblock and there is no text/html part, then no error occurs. This revision would instead generate an error, which is probably wrong. 

I think if we want to start removing try/catch blocks, we're first going to need to remove all of the intentional `throw` statements, and replace them with error codes. To me, that feels more substantial, and probably part of a separate PR. Instead, I'd just do this for this PR:

```cpp
void TrayManager::onClipboardCapture()
{
    const QMimeData *mimeData = QApplication::clipboard()->mimeData();
    QString path;
    QString type;
    if (mimeData->hasHtml() || mimeData->hasText()) {
        QString clipboardContent = mimeData->text();
        if (clipboardContent.isEmpty())
            return;
        Codeblock evidence(clipboardContent);
        try {
            Codeblock::saveCodeblock(evidence);
        }
        catch(FileError& e) {
            QTextStream(stdout) << "Error Gathering Evidence from clipboard" << Qt::endl;
            return;
        }
        path = evidence.filePath();
        type = Codeblock::contentType();
    } else if (mimeData->hasImage()) {
        path  = QDir::toNativeSeparators(SystemHelpers::pathToEvidence().append(Screenshot::mkName()));
        QImage img = qvariant_cast<QImage>(mimeData->imageData());
        img.save(path);
        type = Screenshot::contentType();
    }
    else {
        return;
    }

    try {
        auto evidenceID = createNewEvidence(path, type);
    }
    catch (QSqlError& e) {
      std::cout << "could not write to the database: " << e.text().toStdString() << std::endl;
      return;
    }
    spawnGetInfoWindow(evidenceID);
}
```

**Review by JoelAtDeluxe — COMMENTED** [review]
This seems a touch dangerous. I don't think errors are _likely_, but they're possible. I think it's best to stick to the simple refactoring, and address the try/catches in a later PR when we know we won't be throwing any exceptions.

File: src/traymanager.cpp (hunk 1 above)
Comment (crizzitello) [reply src/traymanager.cpp]: did something similar to this

**Review by JoelAtDeluxe — APPROVED** [review]
Looks better

**Pull request merged**